source/_default_sidebar.md.erb

```diff
@@ -14,15 +14,14 @@ Older [meetings](/meetings/) \| [nights](/nights/) \| [podcasts](/podcasts/)
 
 <%= meeting_calendar_link %>
 
-### Get In Touch
+### Community
 
 * [Mailing List](http://lrug.org/mailing-list)
 * [IRC](/irc/) (#lrug on freenode)
 * [El Rug (Twitter)](http://twitter.com/lrug)
 * Photos: [group (preferred)](http://flickr.com/groups/680991@N25/) or [tag](http://flickr.com/photos/tags/lrug/)
 * [Github](http://github.com/lrug)
-* [Last.FM Group](http://www.last.fm/group/LRUG)
-* [LinkedIn Group](http://www.linkedin.com/e/gis/117315/0027271A7866) (this links to a join request)
```

Comment:
Interesting, there _is_ a linked in group, but it's unlisted and we don't really pay attention to it.  Wonder if we should do a better job of that.  Linked in says it's https://www.linkedin.com/groups/117315/ - but I guess you need to be signed in to see it, and I already am a member so I can't test it properly.  Would you mind checking this url @MikeRogers0 and if it works, we can replace this invite 404 with that link.

Reply:
<img width="385" alt="image" src="https://user-images.githubusercontent.com/325384/91541670-9cf2d880-e914-11ea-88db-860b820d8eb8.png">

Updated. I don't have LinkedIn (or facebook), but I got someone who does to check it for me.